src/components/views/messages/MessageActionBar.js

```diff
@@ -105,14 +112,24 @@ export default class MessageActionBar extends React.PureComponent {
 
     static contextType = RoomContext;
 
+    constructor(props) {
+        super(props);
+        this.elRef = React.createRef();
+        this.state = { focused: "options", buttonNames: [] };
+    }
+
     componentDidMount() {
         this.props.mxEvent.on("Event.decrypted", this.onDecrypted);
         this.props.mxEvent.on("Event.beforeRedaction", this.onBeforeRedaction);
+
+        this.onKeyDown = this.onKeyDown.bind(this);
```

Comment:
you don't need this as you used an arrow func which will have the right `this` context

Reply:
Ah, got it. Still haven't really learned the semantics of class fields